tools/include/pass.hpp

```diff
@@ -81,16 +81,19 @@ void module_pass_manager_apply(const T& x, module_pass_manager& mpm)
 
 } // namespace detail
 
-<%
-interface('pass',
-    virtual('name', returns='std::string', const=True),
-    virtual('apply', returns='void', mpm='module_pass_manager &', const=True, default='migraphx::detail::module_pass_manager_apply'),
-    virtual('apply', returns='void', p='program &', const=True, default='migraphx::nop')
-)
-%>
+<%interface(
+    'pass',
+    virtual('name', returns = 'std::string', const = True),
+    virtual('apply',
+            returns = 'void',
+            mpm     = 'module_pass_manager &',
+            const   = True,
+            default = 'migraphx::detail::module_pass_manager_apply'),
+    virtual('apply', returns = 'void', p = 'program &', const = True, default = 'migraphx::nop'))%>
```

Comment:
Please revert that change, and do not run clang-format on this file - it does not know how to format it correctly because it is a file template and not the regular header file.

Reply:
Thanks for suggestion, I changed it.